cmd/juju/cloud/listcredentials.go

```diff
@@ -383,11 +366,12 @@ func formatCredentialsTabular(writer io.Writer, value interface{}) error {
 		return errors.Errorf("expected value of type %T, got %T", credentials, value)
 	}
 
-	if len(credentials.Client) == 0 {
-		fmt.Fprintln(writer, "No credentials from this client to display.")
-	}
-	if len(credentials.Controller) == 0 {
-		if !credentials.ClientOnly {
+	if credentials.ClientOnly {
```

Comment:
I'm unclear why the change from credentials.Client to credentials.ClientOnly

Reply:
Here .Client has the list of credentials whereas .ClientOnly holds whether the client list was requested or not. The difference is in principal and is needed in the formatter when we get an empty list - is it that there are no credentials on the client or is it that we were not even looking on the client for credentials.

Reply:
I've added comments to clarify.